assets/css/main.scss

```diff
@@ -0,0 +1,12 @@
+---
+---
+
+@import "colors.scss";
+@import "typography.scss";
+@import "nav.scss";
+
+* {
+  box-sizing: border-box;
```

Comment:
This basically makes CSS sizing sane for people not accustomed to width/height calculations involving padding/margins.